src/platform/plugins/shared/discover/public/application/main/components/top_nav/discover_topnav_inline.tsx

```diff
@@ -39,13 +36,6 @@ export const DiscoverTopNavInline = ({
       css={{ boxShadow: 'none', backgroundColor: euiThemeVars.euiPageBackgroundColor }}
       data-test-subj="discoverTopNavInline"
     >
-      {customizationContext.inlineTopNav.showLogsExplorerTabs && (
```

Comment:
This prop can likely be removed now that O11y no longer needs it, but we can handle that as a followup.